app/views/users/_user.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= link_to user_path(user) do %>
+<li class= "user ml-2 mt-4 flex items-center gap-12">
+<div class = "user-image">
+<%= image_tag user.photo, alt: "#{user.name}-avatar", class:'w-40 h-36 object-cover object-center rounded-[50%]' %>
+</div>
+<div class="user-info w-12/12 border border-red flex flex-col">
+<h3 class='username text-3xl ml-2'><%= user.name %>
+<span class='self-end mr-2'><%= user.posts_counter%></span>
+</div>
+</li>
+<%end%>
```

Comment:
- [ ] Kindly style your app layout to match the wireframe.

  Your index page | Wireframe
  :---:|:---:
  ![iis](https://user-images.githubusercontent.com/36542986/230015050-0dfe9c6f-2d3f-42d0-948a-a23b4f1eff31.jpg) | ![jduds](https://user-images.githubusercontent.com/36542986/230015042-2a8fe7a9-be1d-4bc2-8754-63bcac9eebc5.jpg)